### PR TITLE
feat: adds focalboard

### PR DESCRIPTION
**Pull request by codefriar**
adds Focalboard
a trello/notion/asana like tool

### CODE REVIEW

**Comment by codefriar** [comment]
closes #368

File: roles/focalboard/templates/docker-compose.focalboard.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  traefik_network:
+    external:
+      name: vivumlab_traefik
+  focalboard_network:
+
+services:
+  focalboard:
+    restart: unless-stopped
+    # REQUIRED
+    image: mattermost/focalboard
```
Comment (denis-ev) [inline roles/focalboard/templates/docker-compose.focalboard.yml.j2:16]:
```suggestion
    image: mattermost/focalboard:{{ focalboard.version }}
```

File: website/docs/software/focalboard.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+![amd64](https://img.shields.io/badge/{% if not focalboard.amd64 %}untested{% else %}{{ focalboard.amd64 }}{% endif %}-amd64-{% if not focalboard.amd64 %}inactive{% elif focalboard.amd64 == "verified" %}success{% elif focalboard.amd64 == "supported" %}informational{% elif focalboard.amd64 == "unsupported" %}critical{% endif %}?style=flat) <br />
+![arm64](https://img.shields.io/badge/{% if not focalboard.arm64 %}untested{% else %}{{ focalboard.arm64 }}{% endif %}-arm64-{% if not focalboard.arm64 %}inactive{% elif focalboard.arm64 == "verified" %}success{% elif focalboard.arm64 == "supported" %}informational{% elif focalboard.arm64 == "unsupported" %}critical{% endif %}?style=flat) <br />
+![armv7](https://img.shields.io/badge/{% if not focalboard.armv7 %}untested{% else %}{{ focalboard.armv7 }}{% endif %}-armv7-{% if not focalboard.armv7 %}inactive{% elif focalboard.armv7 == "verified" %}success{% elif focalboard.armv7 == "supported" %}informational{% elif focalboard.armv7 == "unsupported" %}critical{% endif %}?style=flat) <br />
+
+## Information
+
+**Docker Image:** !!! LINK TO DOCKER IMAGE/ DOCKER HUB !!! <br />
```
Comment (denis-ev) [inline website/docs/software/focalboard.md:25]:
```suggestion
**Docker Image:** mattermost/focalboard <br />
```

**Comment by codefriar** [comment]
I accepted both your suggestions

**Review by denis-ev — APPROVED** [review]

**Pull request merged**